subsys/net/lib/nrf_provisioning/src/nrf_provisioning.c

```diff
@@ -625,6 +625,8 @@ int nrf_provisioning_req(void)
 			LOG_ERR("Invalid exchange");
 		} else if (ret == -ECONNREFUSED) {
 			LOG_ERR("Connection refused, client exits");
+			LOG_WRN("Please check the CA certificate stored in sectag %d",
+				CONFIG_NRF_PROVISIONING_ROOT_CA_SEC_TAG);
```

Comment:
```suggestion
			LOG_WRN("Please check the CA certificate stored in sectag "
		CONFIG_NRF_PROVISIONING_ROOT_CA_SEC_TAG(CONFIG_NRF_PROVISIONING_ROOT_CA_SEC_TAG));
```

Nitpick... (also cannot get it to line correctly), but you could use STRINGIFY() to just show the value of static configuration macro. It does not change in runtime, so it does not need %d processing.

Reply:
Changed